8.4/Dockerfile.rhel10

```diff
@@ -0,0 +1,75 @@
+FROM ubi10/s2i-core:latest
+
+# MySQL image for OpenShift.
+#
+# Volumes:
+#  * /var/lib/mysql/data - Datastore for MySQL
+# Environment:
+#  * $MYSQL_USER - Database user name
+#  * $MYSQL_PASSWORD - User's password
+#  * $MYSQL_DATABASE - Name of the database to create
+#  * $MYSQL_ROOT_PASSWORD (Optional) - Password for the 'root' MySQL account
+
+ENV MYSQL_VERSION=8.4 \
+    MYSQL_SHORT_VERSION=84 \
+    APP_DATA=/opt/app-root/src \
+    HOME=/var/lib/mysql \
+    NAME=mysql
+
+ENV SUMMARY="MySQL 8.4 SQL database server" \
+    DESCRIPTION="MySQL is a multi-user, multi-threaded SQL database server. The container \
+image provides a containerized packaging of the MySQL mysqld daemon and client application. \
+The mysqld server daemon accepts connections from clients and provides access to content from \
+MySQL databases on behalf of the clients."
+
+LABEL summary="$SUMMARY" \
+      description="$DESCRIPTION" \
+      io.k8s.description="$DESCRIPTION" \
+      io.k8s.display-name="MySQL $MYSQL_VERSION" \
+      io.openshift.expose-services="3306:mysql" \
+      io.openshift.tags="database,mysql,mysql$MYSQL_SHORT_VERSION,mysql-$MYSQL_SHORT_VERSION" \
+      com.redhat.component="$NAME-$MYSQL_SHORT_VERSION-container" \
+      name="rhel10/$NAME-$MYSQL_SHORT_VERSION" \
+      version="1" \
+      com.redhat.license_terms="https://www.redhat.com/en/about/red-hat-end-user-license-agreements#rhel" \
+      usage="podman run -d -e MYSQL_USER=user -e MYSQL_PASSWORD=pass -e MYSQL_DATABASE=db -p 3306:3306 rhel10/$NAME-$MYSQL_SHORT_VERSION" \
+      maintainer="SoftwareCollections.org <sclorg@redhat.com>"
+
+EXPOSE 3306
+
+# This image must forever use UID 27 for mysql user so our volumes are
+# safe in the future. This should *never* change, the last test is there
+# to make sure of that.
+RUN INSTALL_PKGS="policycoreutils rsync tar gettext hostname bind-utils groff-base mysql-server procps-ng" && \
```

Comment:
`mysql-server` does not exist on RHEL 10 / C10S. (need fixing in both)
Only `mysql8.4-server` does.

The unversioned names are no longer provided ([RHEL-68750](https://issues.redhat.com/browse/RHEL-68750))

Reply:
This was fixed by commit https://github.com/sclorg/mysql-container/pull/388/commits/5aa8a9f59263b33bcabab5882d4ca40cc2bd5166